From 1f22dc6be768b7032b73ea963901de270e3c99d9 Mon Sep 17 00:00:00 2001
From: Lans Zhang <jia.zhang@windriver.com>
Date: Sun, 18 Jun 2017 21:50:26 +0800
Subject: [PATCH] netboot: replace the depreciated EFI_PXE_BASE_CODE

The newer gnu-efi already uses EFI_PXE_BASE_CODE_PROTOCOL instead.

Signed-off-by: Lans Zhang <jia.zhang@windriver.com>
---
 netboot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/netboot.c b/netboot.c
index 1cc1a2b..32c9b0a 100644
--- a/netboot.c
+++ b/netboot.c
@@ -43,7 +43,7 @@
 #define ntohs(x) __builtin_bswap16(x)	/* supported both by GCC and clang */
 #define htons(x) ntohs(x)
 
-static EFI_PXE_BASE_CODE *pxe;
+static EFI_PXE_BASE_CODE_PROTOCOL *pxe;
 static EFI_IP_ADDRESS tftp_addr;
 static CHAR8 *full_path;
 
-- 
2.7.5

